cognite/client/_api/entity_matching.py

```diff
@@ -109,7 +128,11 @@ def list(
             limit (int, optional): Maximum number of items to return. Defaults to 100. Set to -1, float("inf") or None to return all items.
 
         Returns:
-            EntityMatchingModelList: List of models."""
+            EntityMatchingModelList: List of models.
+
+        Examples:
+                >>> client.entity_matching.list(limit=1, name="test")
```

Comment:
seems the errors are complaining that client and the import for client does not exist?